input-forms.md

```diff
@@ -0,0 +1,82 @@
+# Input-forms Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/input-forms**   
+
+Provide access to the configured input-forms. It returns the list of existent input-forms.
+Please note that starting from DSpace 7 each input-form will describe a single page of inputs, combining different pages togheter is done with the [submission-definition](submission-panels.md), aka the item-submission.xml configuration file 
+
+Example: to be provided
+
+## Single Input-Form 
+**/api/configuration/input-forms/<:form-name>**
```

Comment:
If these are being redefined as pages, I wonder whether we should even call them `input-forms` any longer.  Why not rename this endpoint to be `submission-page` or `submission-form`?

Reply:
@tdonohue see https://github.com/DSpace/Rest7Contract/pull/11#discussion_r144770889 for your first comment
I like the idea to rename them in submission-form